app/mutations/application/createApplicationStatusMutation.ts

```diff
@@ -0,0 +1,32 @@
+import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {
+  createApplicationStatusMutationVariables,
+  createApplicationStatusMutation as createApplicationStatusMutationType
+} from 'createApplicationStatusMutation.graphql';
+import BaseMutation from '../BaseMutation';
+
+const mutation = graphql`
+  mutation createApplicationStatusMutation(
+    $input: CreateApplicationStatusInput!
+  ) {
+    createApplicationStatus(input: $input) {
+      clientMutationId
+      applicationStatus {
+        id
```

Comment:
As discovered this week, the reason the application status isn't updating in the page is because this mutation response is incomplete. The following needs to be added here:
```
applicationByApplicationId {
  id
  applicationStatus {
    id
    applicationStatus
  }
}
```
This will update the `applicationStatus` field in the `application` with the newly created `applicationStatus`, and should update what is rendered without requiring a refresh

Reply:
TODO after #237 gets merged